README.md

```diff
@@ -1,2 +1,110 @@
 # tao-e2e-runner
-End to end test runner 
+
+End to end test runner.
+
+The project is based on Cypress (https://www.cypress.io/)
+
+## NPM Commands:
+
+Run all test:
+
+```
+npm run test
+```
+
+Run specific test:
+
+```
+npm run test <testname>
+```
+
+Run tests on other domain:
+
+_Default: http://tao.docker.localhost_
+
+```
+CYPRESS_baseUrl=http://example.com npm run test
+```
+
+Run tests from other location:
+
+_Default: exampleTests_
+```
+CYPRESS_integrationFolder=exampleTests npm run test
+```
+
+Modify pattern of test files:
+
+_Default: **/e2e/*.spec.js_
+
+```
+CYPRESS_testFiles="**/e2e/*.spec.js" npm run test
+```
+
+Open Cypress interface
+
+```
+npx cypress open
+```
+
+## Configs:
+
+### Environment variables:
+
+You can use environment variables in your tests, that can be defined as environment variable like:
+
+```
+CYPRESS_foo=bar npm run test
+```
+
+and you can use it in your test like:
+
+```js
+const foo = Cypress.env('foo');
+```
+
+You can also define environment variables with config file that should be called **cypress.env.json**. Testrunner supports the following variables:
```

Comment:
It would be very useful to be able to define the `baseUrl` also as par of `cypress.env.json`

Reply:
I wrote a plugin for that, so it is working from now.